proposals/0275-allow-more-characters-like-whitespaces-and-punctuations-for-escaped-identifiers.md

```diff
@@ -14,19 +14,39 @@
 Swift has a beautiful concise yet expressive syntax.
 As part of that, escaped identifiers are adopted to allow usage of reserved keywords.
 This proposal wants to extend the character allowance for escaped identifiers with more Unicode scalars, like whitespace and punctuation.
-It will enable to have method names (or other identifiers) with a more readable and natural language like the following:
-```swift
-func `test validation should succeed when input is less than ten`()
-```
+
 ## Motivation
+
 Naming could be hard and having descriptive methods, like in tests, may result in declarations that are hard to read because of its lack of whitespace and punctuations or other symbols. Enabling natural language would improve readability.
 
+```swift
+func `test validation should succeed when input is less than ten`() // currently not possible
+// vs
+func `testValidationShouldSucceedWhenInputIsLessThanTen`()
+```
+
```

Comment:
I think you should also add these examples:

```swift
func test_Validation_Should_Succeed_When_Input_IsLess_Than_Ten() {}
func test_validationShouldSucceed_whenInputIs_lessThanTen() {}
```
(add a few more variations if needed)

for a fairer comparison, because this is what a lot of people (including myself) currently write test functions as (and I think they are quite readable as well).